agml/synthetic/manual.py

```diff
@@ -101,7 +102,7 @@ def generate_manual_data(code = None,
                         f"could not be found. Check that Helios is installed there.")
 
         # Check that the necessary files exist.
-        if not os.path.exists(os.path.join(project_path, 'main.cpp')):
+        if not os.path.exists(os.path.join(project_path, 'main.cpp')) and not os.path.exists(os.path.join(project_path, 'generate.cpp')):
```

Comment:
This change (and additional changes below) allows the Helios project code to be named `generate.cpp` in addition to `main.cpp`, which makes the `generate_manual_data` slightly more flexible.